src/components/MapView/Layers/AdminLevelDataLayer/index.tsx

```diff
@@ -28,15 +28,21 @@ import {
 import { getRoundedData } from '../../../../utils/data-utils';
 import { useSafeTranslation } from '../../../../i18n';
 import { fillPaintData } from '../styles';
+import { availableDatesSelector } from '../../../../context/serverStateSlice';
```

Comment:
@JorgeMartinezG can you review this fix for the validity period and confirm it makes sense to you?